Getting started documentation

DmitriyNoa opened the pull request.

arjunu commented on docs/getting-started.md:
```diff
@@ -0,0 +1,43 @@
+1. NodeJS version >=10.0.0 <12.0.0.
+2. [Yarn](https://yarnpkg.com/lang/en/).
+
+### Full:
```
should we mention what full install is for? Like what docker and Ganache is required for?

DmitriyNoa replied:
Yes, I will mention this as soon as I find out how to run everything locally with local backend

drakkein replied:
Also it's worth to mention that we have ability to run frontend on remote backend instance `start:remote:vm` task, which requires to have `NF_VM_ADDRESS="ip here"` set in `.env` file.

DmitriyNoa replied:
Yes, but it generates it without those 2 envs variables provided and fails immediately, so you need to stop the process, define vars and start again

arjunu commented:
Root file [README](`https://github.com/Neufund/platform-frontend/blob/master/README.md`) already has some of these steps. Will you merge these files?

drakkein commented on docs/getting-started.md:
`````diff
@@ -0,0 +1,43 @@
+4. Generate contract artifacts ````yarn contract-artifacts:generate````.
+5. Setup environment variables:
+- Make sure that .env file is in place ./packages/web/.env
+- If it's not create it ````touch .env````.
`````
Also `.env` should be created automatically from `.env.example` while running any `start *` task

arjunu replied:
Yeah I don't rem creating it myself

DmitriyNoa replied:
Yes, but it generates it without those 2 envs variables provided and fails immediately, so you need to stop the process, define vars and start again

drakkein commented:
@DmitriyNoa If you don't have it yet, you can ask @pstroinski about remote VM to run backend on it

DmitriyNoa commented:
I moved the getting started docs to the main README in the web package. The file already contained some information so I moved the missing parts to it. Please check

arjunu commented on packages/web/README.md:
```diff
@@ -86,3 +130,26 @@ yarn lint:fix
 
 job
+
+## Possible issues:
```
👍

arjunu commented on packages/web/README.md:
```diff
@@ -9,12 +9,48 @@ A web UI of neufund platform.
+5. Setup environment variables:
+
+- Make sure that .env file is in place ./packages/web/.env.
+- If it's not create it run `yarn start:remote`.
```
```suggestion
- If it's not, create it and run `yarn start:remote`.
```

arjunu commented on packages/web/README.md:
```diff
@@ -9,12 +9,48 @@ A web UI of neufund platform.
+1. NodeJS version >=10.0.0 <12.0.0.
+2. [Yarn](https://yarnpkg.com/lang/en/).
+
+## Setting environment.
```
```suggestion
## Setting up the environment
```

arjunu commented:
There are some sectioning issues in the file:
- Uploading contracts is under section "Running project"
", should be in a different one
- "Fix all autofixable errors" and "Production build" sections are under "Running tests", instead should be separate
- "Extracting translations from the app" and "Update contract artifacts submodule" are under the section "Running storybook" , should be in a different one
- There's a typo "job" above "Possible issues" title
